projects/plugins/boost/app/assets/src/js/pages/purchase-success/purchase-success.tsx

```diff
@@ -3,17 +3,18 @@ import { Button } from '@wordpress/components';
 import { createInterpolateElement, useEffect } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { requestImageAnalysis } from '$features/image-size-analysis/lib/stores/isa-summary';
-import enableCloudCss from '$lib/utils/enable-cloud-css';
 import { navigate } from '$lib/utils/navigate';
 import Logo from '$svg/jetpack-green';
+import { useModuleState } from '$features/module/lib/stores';
 
 type PurchaseSuccessProps = {
 	isImageGuideActive: boolean;
 };
 
 const PurchaseSuccess: React.FC< PurchaseSuccessProps > = ( { isImageGuideActive } ) => {
 	useEffect( () => {
-		enableCloudCss();
+		const [ , setCloudCssState ] = useModuleState( 'cloud_css' );
+		setCloudCssState( true );
```

Comment:
You can't call a `use*` React hook inside a `useEffect`. It must be called directly from the component's function.

Reply:
https://github.com/Automattic/jetpack/pull/34662/commits/3577f2454c8ffcb896a9dfb6d9a8ce8e500d4f89